blazor/accordion/accessibility.md

```diff
@@ -32,4 +58,14 @@ Keyboard navigation is enabled by default. The possible keys are:
 | <kbd>Down Arrow</kbd>   | Focus the next Accordion header. |
 | <kbd>Up Arrow</kbd>         | Focus the previous Accordion header. |
 | <kbd>Home</kbd>           | Focus the first Accordion header. |
-| <kbd>End</kbd>   | Focus the last Accordion header. |
\ No newline at end of file
+| <kbd>End</kbd>   | Focus the last Accordion header. |
+
+## Ensuring accessibility
+
+The Accordion component's accessibility levels are ensured through an [accessibility-checker](https://www.npmjs.com/package/accessibility-checker) and [axe-core](https://www.npmjs.com/package/axe-core) software tools during automated testing.
+
+The accessibility compliance of the Accordion component is shown in the following sample. Open the `sample` in a new window to evaluate the accessibility of the Accordion component with accessibility tools.
```

Comment:
Find Sample link here.

https://github.com/essential-studio/blazor-accessibility-demos/wiki/Blazor-Accessibility-Live-Samples-Link-%E2%80%90-AxeCore

Reply:
Hi @rajendranr-5483 ,

I added the sample link,

Regards,
@Backiaraj